src/test/php/rdbms/unittest/integration/RdbmsIntegrationTest.class.php

```diff
@@ -715,7 +715,7 @@ public function unbufferedReadOneResult() {
     $this->createTable();
     $db= $this->db();
 
-    $q= $db->open('select * from %c', $this->tableName());
+    $q= $db->query('select * from %c', $this->tableName());
     $this->assertEquals(['pk' => 1, 'username' => 'kiesel'], $q->next());
```

Comment:
This change changes the test (called "unbufferedReadOneResult") to use *buffered* queries. Not sure why you're making this change here, please revert it though as it now tests something different!